src/site/content/en/media/accessibility/index.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Let's also mention this page under https://web.dev/accessible/#understand-semantics-and-basic-screen-reader-support …?

Reply:
Done.